cea/technologies/storage_tank_pcm.py

```diff
@@ -61,8 +61,9 @@ def __init__(self, size_Wh:float, database_model_parameters, T_ambient_K:float,
         # initialize variables and properties (empty storage)
         self.T_tank_K = self.T_tank_fully_discharged_K
         self.current_phase = 1  # "1=liquid, 2=phasechange, 3=solid"
-        self.current_storage_capacity_Wh = 0.0  # since the simualtion start at mid-night it is considered that the storage is half full by then.
-        self.current_thermal_gain_Wh = self.current_storage_capacity_Wh * (1.05 ** (self.hour - self.hour_of_last_activation) - 1)  # Assumption of losses of X% when discharged during this hour. TODO: Check this value with an example
```

Comment:
There seems to be a 5% factor (losses?) here which is not considered by your changes. Was that intentional?

Reply:
Yes, that was intentional. I mistakenly added these statements to the class description a while back, since I thought it would be called again every time a `storage_tank_pcm` class object is modified, these lines would be triggered again.

The lines replacing this section are [194, 244, 250] when the storage is charged and [267, 310, 316] when the storage is discharged.